project/Dependencies.scala

```diff
@@ -35,7 +35,7 @@ object Dependencies {
   val logbackClassic: ModuleID = "ch.qos.logback"             %  "logback-classic" % "1.2.3"
   val ravenLogback: ModuleID =   "com.getsentry.raven"        %  "raven-logback"   % "7.8.6"
   val scalaLogging: ModuleID =   "com.typesafe.scala-logging" %% "scala-logging"   % scalaLoggingV
-  val swaggerUi: ModuleID =      "org.webjars"                %  "swagger-ui"      % "3.52.3"
+  val swaggerUi: ModuleID =      "org.webjars"                %  "swagger-ui"      % "4.1.2"
```

Comment:
Thought I'd bump swagger to the latest -- it seems to work fine, there aren't any breaking changes.

https://swagger.io/blog/news/what-s-new-in-swaggerui-v4-and-swaggereditor-v4/